Makefile

```diff
@@ -507,20 +508,20 @@ set-image-ubi: kustomize copy-config ## Set the controller UBI image
 	cd $(CONFIG_MANAGER_DIR) && $(KUSTOMIZE) edit set image controller=$(IMG_UBI)
 
 .PHONY: sdk-generate
-sdk-generate: copywrite
-	operator-sdk generate kustomize manifests -q
+sdk-generate: copywrite operator-sdk
+	$(OPERATOR_SDK) generate kustomize manifests -q
 	@$(COPYWRITE) headers &> /dev/null
 
 .PHONY: bundle
-bundle: manifests kustomize sdk-generate set-image-ubi yq ## Generate bundle manifests and metadata, then validate generated files.
+bundle: manifests kustomize sdk-generate set-image-ubi yq operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
```

Comment:
This is out of my curiosity, not my actual code review :). 

I see `operator-sdk` is added to `sdk-generate`, and `sdk-generate` is added to `manifests`. I'm curious how `manifests` in `bundle` is executed differently that `sdk-generate` and `operator-sdk` remain in `bundle` (i.e why we don't include only `manifests` calling `sdk-generate` which also calls `operator-sdk`).